diff -urN mercurial.opensound.com/setup/srcconf.c mercurial.opensound.com-new/setup/srcconf.c
--- mercurial.opensound.com/setup/srcconf.c	2009-06-21 10:03:24.000000000 +0000
+++ mercurial.opensound.com-new/setup/srcconf.c	2009-06-21 10:17:51.000000000 +0000
@@ -1170,7 +1170,7 @@
 
       fprintf (f, "$(LIBDIR)/%s.so:\t$(OBJECTS)\n", name);
       fprintf (f,
-	       "\t$(LD) $(LDFLAGS) %s -o $(LIBDIR)/%s.so $(OBJECTS)\n",
+	       "\t$(LD) $(raw-ldflags) %s -o $(LIBDIR)/%s.so $(OBJECTS)\n",
 	       shlib_ldflags, name);
       fprintf (f, "\n\n");
     }
