diff -urN phc-0.2.0.3.orig/Makefile.am phc-0.2.0.3/Makefile.am
--- phc-0.2.0.3.orig/Makefile.am	2009-03-14 05:55:00.000000000 +0000
+++ phc-0.2.0.3/Makefile.am	2009-03-14 05:56:50.000000000 +0000
@@ -535,7 +535,7 @@
 
 # - Without the second \@, autoconf with replace the @CFLAGS\@ etc
 src/phc_compile_plugin: Makefile src/phc_compile_plugin.in
-	sed -e 's,@includedir\@,$(includedir),g; s,@CXX\@,$(CXX),g; s,@CFLAGS\@,$(CFLAGS),g; s,@LDFLAGS\@,$(LDFLAGS),g; s,@libexecdir\@,$(libexecdir),g; s,@pkglibdir\@,$(pkglibdir),g;' $(srcdir)/src/phc_compile_plugin.in > src/phc_compile_plugin
+	sed -e 's,@includedir\@,$(includedir),g; s,@CXX\@,$(CXX),g; s,@CFLAGS\@,$(CFLAGS),g; s!@LDFLAGS\@!$(LDFLAGS)!g; s,@libexecdir\@,$(libexecdir),g; s,@pkglibdir\@,$(pkglibdir),g;' $(srcdir)/src/phc_compile_plugin.in > src/phc_compile_plugin
 	chmod 755 src/phc_compile_plugin
 
 test/framework/lib/autovars.php: Makefile test/framework/lib/autovars.php.in
diff -urN phc-0.2.0.3.orig/Makefile.in phc-0.2.0.3/Makefile.in
--- phc-0.2.0.3.orig/Makefile.in	2009-03-14 05:55:00.000000000 +0000
+++ phc-0.2.0.3/Makefile.in	2009-03-14 10:22:32.000000000 +0000
@@ -2818,7 +2818,7 @@
 
 # - Without the second \@, autoconf with replace the @CFLAGS\@ etc
 src/phc_compile_plugin: Makefile src/phc_compile_plugin.in
-	sed -e 's,@includedir\@,$(includedir),g; s,@CXX\@,$(CXX),g; s,@CFLAGS\@,$(CFLAGS),g; s,@LDFLAGS\@,$(LDFLAGS),g; s,@libexecdir\@,$(libexecdir),g; s,@pkglibdir\@,$(pkglibdir),g;' $(srcdir)/src/phc_compile_plugin.in > src/phc_compile_plugin
+	sed -e 's,@includedir\@,$(includedir),g; s,@CXX\@,$(CXX),g; s,@CFLAGS\@,$(CFLAGS),g; s!@LDFLAGS\@!$(LDFLAGS)!g; s,@libexecdir\@,$(libexecdir),g; s,@pkglibdir\@,$(pkglibdir),g;' $(srcdir)/src/phc_compile_plugin.in > src/phc_compile_plugin
 	chmod 755 src/phc_compile_plugin
 
 test/framework/lib/autovars.php: Makefile test/framework/lib/autovars.php.in
